assets/js/atomic/blocks/component-init.js

```diff
@@ -63,6 +63,15 @@ registerBlockComponent( {
 	),
 } );
 
+registerBlockComponent( {
+	blockName: 'woocommerce/product-average-rating',
+	component: lazy( () =>
+		import(
+			/* webpackChunkName: "product-average-rating" */ './product-elements/average-rating/block'
+		)
```

Comment:
I am getting TS error on these lines, as you can see below:
<img width="1208" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/3ff06cf2-f057-4bc7-9504-bd9593756ee3">